docs/devel/e2e-tests.md

```diff
@@ -208,13 +208,31 @@ We are working on implementing clearer partitioning of our e2e tests to make run
 
 ### Conformance tests
 
-Finally, `[Conformance]` tests are tests we expect to pass on **any** Kubernetes cluster.  The `[Conformance]` label does not supersede any other labels.  `[Conformance]` test policies are a work-in-progress (see #18162).
+Finally, `[Conformance]` tests represent a subset of the e2e-tests we expect to pass on **any** Kubernetes cluster.  The `[Conformance]` label does not supersede any other labels.
+
+As each new release of Kubernetes providers new functionality, the subset of tests necessary to demonstrate conformance grows with each release.  Conformance is thus considered versioned with an eye towards backwards compatibility.  Conformance tests for a given version should be run off of the release branch that corresponds to that version.  Thus `v1.2` conformance tests would be run from the head of the `release-1.2` branch. eg:
```

Comment:
do we have a limit on how long we expect conformance tests to work? I think the upgrade policy is trailing 3 versions (or something like that).